src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -1124,10 +1126,10 @@ sub check_star_catalog {
 	# Faint and bright limits ~ACA-009 ACA-010
 	if ($type ne 'MON' and $mag ne '---') {
 
-	    if ($mag < $mag_bright or $mag > $mag_faint_red) {
+	    if ($mag < $mag_bright or $mag > $self->{mag_faint_red}) {
```

Comment:
I think that we decided to put in the probabilities block (or somewhere persistent anyway).  That way you always know the values even if there are no warnings.